lean4-infoview/package.json

```diff
@@ -6,7 +6,7 @@
         "watch": "rollup --config --environment NODE_ENV:development --watch",
         "build": "rollup --config --environment NODE_ENV:production",
         "watchTest": "npm run watch",
-        "test": "tsc -p test/tsconfig.json"
+        "test": "./test/run-tests.sh"
```

Comment:
It should be possible to run the tests on Windows.

Reply:
I moved that to an independent PR